mpf/devices/state_machine.py

```diff
@@ -52,7 +52,7 @@ def validate_and_parse_config(self, config: dict, is_mode_config: bool, debug_pr
     @property
     def state(self):
         """Return the current state."""
-        if self.config['persist_state']:
```

Comment:
This corrects a behavior in MPF Monitor, which attempts to initialize all state_machines on bootup (for displaying in the device list). Without a player, Monitor's attempt to get the state of the machine causes a crash.

Reply:
Good catch. We should probably add some integration test for this so that this will not happen again for other devices. We already had that a few times.